src/components/Link/index.js

```diff
@@ -13,11 +13,13 @@ const A = styled.a`
 
     ${props => props.isGreen
         && css`
-            border-bottom: 1px solid ${colors.GREEN_PRIMARY};
+            text-decoration: underline;
+            text-decoration-color: ${colors.GREEN_PRIMARY};
```

Comment:
text-decoration-color is unnecessary.
There should also be a hover efect with text-decoration: none; on line 40

Reply:
let's take it a step further and make it underline by default.